pkg/loopback/loopback_test.go

```diff
@@ -0,0 +1,136 @@
+// +build linux,cgo
+
+package loopback
+
+import (
+	"flag"
+	"io/ioutil"
+	"os"
+	"strings"
+	"syscall"
+	"testing"
+
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const filePath = "/tmp/"
+
+var rootEnabled bool
+
+func init() {
+	flag.BoolVar(&rootEnabled, "test.root", false, "enable tests that require root")
```

Comment:
> It is about the potential for destruction as a root-level test.

What is a "root-level test" ? Do you mean a test that requires root to run? If a user is running tests as root, they are already assuming that risk. 

> Test flags are easy to build and add and are self-documenting, since they show in the help of a test.

Do you really think that running `--help` on a test binary is a common use case? The default `go test` doesn't even expose the binary.